### PR TITLE
Skip clamav locally

### PR DESCRIPTION
**Pull request by abottoms-coder**
## Summary of Changes
_Provide a brief summary of changes_
To improve spin-up time, I have set an environment flag for the Django side of our data file scans (CLAMAV_NEEDED). As a TODO, I need to potentially address the conditional spin-up of clamav in docker compose.

## How to Test
_List the steps to test the PR_

1. Setup environment variables, load aliases, and clean docker caches before new build/start.
```bash
export TDRS_HOME=`pwd`
source commands.sh
tdrs-prune-all-docker-data
echo "CLAMAV_NEEDED=False" >> tdrs-backend/.env
export CLAMAV_NEEDED="False"
tdrs-start
```
1. Using `docker ps`, ensure neither ZAP nor CLAMAV images were created or spun up

1. Open http://localhost:3000/ and sign in.
1. Ensure there are no redirect loops.
1. Ensure 
> *Demo GIF(s) and screenshots for testing procedure*
image

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] Prevent unnecessary waiting and spinup of CLAMAV and ZAP unless developer needs.
+ [x] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces backend code changes, are they meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_93.39%_] (see `CodeCov Report` comment in PR)
  + [x] Backend coverage: [_97%_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

_n/a_

### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

_n/a_

### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

_n/a_

### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

_n/a_

### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

_n/a_

### CODE REVIEW

File: tdrs-frontend/docker-compose.local.yml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-version: "3.9"
+version: "3.4"
 
 services:
   tdp-frontend:
     stdin_open: true # docker run -i
     tty: true        # docker run -t
     build:
       context: .
-      target: nginx
```
Comment (abottoms-coder) [inline tdrs-frontend/docker-compose.local.yml:11]: I think we copied this from the regular docker-compose.yml which resulted in the loop.

File: scripts/set-backend-env-vars.sh [hunk 2]
```diff
@@ -40,6 +40,8 @@ cf set-env "$CGAPPNAME_BACKEND" DJANGO_SECRET_KEY "$DJANGO_SECRET_KEY"
 cf set-env "$CGAPPNAME_BACKEND" FRONTEND_BASE_URL "$FRONTEND_BASE_URL"
 cf set-env "$CGAPPNAME_BACKEND" DJANGO_CONFIGURATION "$DJANGO_CONFIGURATION"
 cf set-env "$CGAPPNAME_BACKEND" DJANGO_SETTINGS_MODULE "$DJANGO_SETTINGS_MODULE"
+cf set-env "$CGAPPNAME_BACKEND" CLAMAV_NEEDED "$CLAMAV_NEEDED"
```
Comment (abottoms-coder) [inline scripts/set-backend-env-vars.sh:43]: Added here to ensure this is set in cloud.gov deployments.

**Review by riatzukiza — APPROVED** [review]

**Review by jorgegonzalez — APPROVED** [review]

**Comment by abottoms-coder** [comment]
This was branched from #1645 so this merge makes that PR obsolete, will close.

**Review by raftmsohani — APPROVED** [review]

**Review by ADPennington — APPROVED** [review]
lgtm 👍🏾

**Pull request merged**